src/dropdown/dropdown.js

```diff
@@ -23,6 +24,7 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
   this.close = function(dropdownScope) {
     if (openScope === dropdownScope) {
       openScope = null;
+      $document.unbind('touch', closeDropdown);
```

Comment:
In order to be more precise, $document.bind is not working only in A tags in mobilephone